src/java/com/puppetlabs/ssl_utils/ExtensionsUtils.java

```diff
@@ -845,6 +850,28 @@ private static Integer getGnTagFromName(String name) {
     }
 
     /**
+     * Convert the value of an IP address which is encoded in an
+     * ASN1OctetString to a string.
+     *
+     * @param ip IP address encoded in an octet string.
+     * @return A string representing the given IP address.
+     */
+    public static String octetStringToIpString(ASN1OctetString ip) {
+        StringBuilder str = new StringBuilder();
+
+        byte[] octets = ip.getOctets();
```

Comment:
Are there any existing helper methods that you could use to parse this.  Like I'm wondering if you sent the byte[] through `java.net.InetAddress.getByAddress.toString` would you get the same thing - or is there some unique decoding for the ASN1 format that you couldn't find another helper for?